src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class SetCollectionTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void size(){
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    void contains(int input) {
+        assertThat(numbers.contains(input)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1,2,3:true", "4,5:false"}, delimiter = ':')
+    void containsList(String input, boolean expect) {
+        List<Integer> list = Arrays.stream(input.split(",")).map(Integer::parseInt).collect(Collectors.toList());
```

Comment:
```suggestion
        List<Integer> list = Arrays.stream(input.split(","))
        .map(Integer::parseInt)
        .collect(Collectors.toList());
```

가독성과 디버깅을 위해 이렇게 해도 좋을듯해요!

Reply:
줄을 나누는게 더 보기 좋네요~ 감사합니다!